src/darwin/agent/launchd.js

```diff
@@ -24,6 +27,16 @@ rpc.exports = {
   disableSpawnGating: function () {
     gating = false;
   },
+  forgetPid: function (pid) {
+    delete suspendedPids[pid];
+  },
+  dispose: function () {
```

Comment:
Would be good to move this first for consistency (the convention followed currently is to always put `dispose` first, or, right after the constructor).